src/app/profiles/hooks/useContributorsFilters.ts

```diff
@@ -24,7 +24,13 @@ function useContributorsFilters(
     );
 
   useEffect(() => {
-    if (selectedCategory && selectedCategory === ALL) {
+    console.log(
```

Comment:
We shouldn't log anything in the content script.
We may log things in the background using the appropriate `logger` service.